packages/recordio/index.js

```diff
@@ -0,0 +1,3 @@
+import read from "./src/read";
+
+export default read;
```

Comment:
Let's change the export to `export {default as read} form "./src/read"` so that we could potentially add a _write_ without breaking imports.

Reply:
Then we first have to readjust our eslint that insists I prefered default export.

Reply:
If we find ourselves adding a write in recordio format, I'd certainly have some questions.

Reply:
Point is, as long as this is a "general" `recordio` and not a `recordio-read` package, we should not prevent ourselves from adding new functions.

Reply:
I'm going to switch to dynamic export for now